src/Neo/SmartContract/Native/NativeContract.cs

```diff
@@ -361,7 +362,7 @@ public static NativeContract GetContract(UInt160 hash)
             return contract;
         }
 
-        internal async void Invoke(ApplicationEngine engine, byte version)
+        internal async Task InvokeAsync(ApplicationEngine engine, byte version)
```

Comment:
I'm worry in line 392, when we check that it must be a `ContractTask` and now some method returns `Task`

Reply:
I already tested this. Plus if it was void then it needs to be task. I also tried `ContractTask` and didn't work.